### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by hitomipupil** [HYF-Class22/agile-development-group3-hyf-redesign#2] [General]
Let's talk about backlog

**Comment by hitomipupil** [comment]
Must-haves

**Reply by hitomipupil** [reply]
- header(Title, logo)
- navigation
- links for users
- Main(photos, testimony, text)
- contact form
- footer(adress, SNS links)

**Comment by hitomipupil** [comment]
Could-Haves

**Reply by hitomipupil** [reply]
- Video in the main
- FAQ
- Google Map

**Comment by hitomipupil** [comment]
Should-Haves

**Comment by Tesfay20** [comment]
hi @hitomipupil , Can you review and merge  the `partnership ` user personas? 

Partnership


Bio: `John ` is a big company owner interested to collaborate with schools offering market oriented specialties and would like to hire graduates with the right qualifications. 
Needs/Goals: `John `is searching for more information on how to increase the tangible and intangible values of his company through collaboration with the schools by offering internships.
Frustrations: `John ` can't find a website which provides clear and concise user information on how to collaborate with schools that promote in producing talented graduates. Most websites offer complex information about collaboration and are too general and do not focus on the practicability of collaboration with the right schools
Scenario 1: `John `is a  company owner  who wants to have partnership with schools and and as a result, wants to hire talented graduates that are familiar to the job requirements in his company.

**Reply by hitomipupil** [reply]
hi @Tesfay20 , Thank you for your great imagination! I think this is good, but could you post it on the [discussion of User Persona](https://github.com/HYF-Class22/agile-development-group3-hyf-redesign/discussions/1) at Partnership thread?

**Comment by Tesfay20** [comment]
Thanks @hitomipupil , done!
